extensions/AtlasMongoDb/AtlasMongoDb/Helpers/AtlasSearchHelper.cs

```diff
@@ -0,0 +1,362 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Driver;
+
+namespace KernelMemory.AtlasMongoDb.Helpers;
+
+/// <summary>
+/// <para>Wrapper for ATLAS search indexes stuff</para>
+/// <para>
+/// <ul>
+/// <li>https://www.mongodb.com/docs/v7.0/reference/method/db.collection.createSearchIndex/</li>
+/// <li>Normalizer (end of the page) https://www.mongodb.com/docs/atlas/atlas-search/analyzers/</li>
+/// </ul>
+/// </para>
+/// </summary>
+public class AtlasSearchHelper
```

Comment:
can we keep this class `internal` only?

Reply:
I added internalsvisible to because it is used by test to cleanup.